src/main/java/com/google/ehub/data/CommentData.java

```diff
@@ -8,11 +8,17 @@
   private final String comment;
   private final Long timestampMillis;
   private final String username;
+  private final Long commentId;
+  private final String commentKind;
+  private final Boolean belongsToUser;
```

Comment:
Does this field need to be Boolean (object wrapper for primitive boolean)?
```suggestion
  private final boolean belongsToUser;
```